### DISCUSSION TITLE
Ads

### DISCUSSION THREAD

**Discussion by paulthetic** [AprilSylph/XKit-Rewritten#246] [Suggestion Box]
Is it possible to let us add features by ourselves in the future? I really need anti-capitalism because I don't know how to block ads with ublock origin without using element zapper each time.

**Comment by AprilSylph** [comment]
First off, you can continue using Anti-Capitalism in XKit 7; the two addons do not inherently conflict.

To enable filter lists in uBlock Origin, open the settings popup (same as you would with this addon), then click the three gears icon at the bottom-right, then go to the "Filter lists" tab. Right now, **AdGuard Base** should block sponsored posts. I'm not sure which filter list it is that blocks the sidebar ad for me (resulting in an empty "Sponsored" sidebar section), but once you're able to configure your filter lists I recommend enabling... pretty much all of them. The only ones I don't have enabled are Dan Pollock's hosts file, MVPS HOSTS, or any of the region/language-specific lists.

**Reply by AprilSylph** [reply]
See also: https://github.com/AprilSylph/XKit-Rewritten/issues/228#issuecomment-913873285
